dependent-service-downtime-and-ps-adaptor-behavior.md

```diff
@@ -0,0 +1,64 @@
+## Service Failures and Expected Behavior
+
+This guide outlines the expected behavior of the PS Adaptor when various dependent services are down. 
+The PS Adaptor works in conjunction with several services including the GP2GP Translator, MHS Adaptor, Facade, and others. 
+The guide explains the effects of outages and how the system behaves in each case.
+
+### System Overview
+The PS Adaptor initiates a conversation for transferring patient records, relying on several interconnected services:
+- GP2GP Translator: Sends patient records to the PS Adaptor via a queue.
+- Facade Application: Works with the PS Adaptor to manage transfer requests.
+- MHS Adaptor: Manages inbound and outbound queues, which are responsible for transferring patient records to the incumbent system.
+- Other Dependencies: Include PS Adaptor database, MHS inbound and outbound services, file storage, Redis cache, and MHS outbound SDS.
+
+### Service Outage Scenarios and Expected Behavior
+1. GP2GP Translator is Down
+   - Scenario: The GP2GP Translator is not operational and cannot send patient records.
+   - Expected Behavior: No messages are sent to the incumbent system while the translator is down. Once the GP2GP Translator recovers, 
+     the transfer process resumes, and messages are processed as normal.
+   - Recovery: Automatic upon Translator recovery; transfer resumes without manual intervention.
+2. Queue is Down
+   - Scenario: The queue responsible for transferring data between the GP2GP Translator and PS Adaptor is down.
+   - Expected Behavior: The initial request is never sent to the incumbent system because the queue is unavailable.
+   - Recovery: After the queue is restored, the request can be processed normally, but any messages queued during downtime are not sent.
+3. PS Adaptor Database (DB) is Down
+   - Scenario: The PS Adaptor's database is not operational.
+   - Expected Behavior: The initial request to the incumbent system is not sent because the adaptor cannot access the necessary data.
+   - Recovery: After the database is restored, the system may resume operations, but requests during the outage are not processed.
```

Comment:
Again I'm confused by the wording "initial request to the incumbent system is not sent"

Reply:
I will rephrase it